pyVim/connect.py

```diff
@@ -532,7 +537,7 @@ def __FindSupportedVersion(protocol, server, port, path, preferredApiVersions):
 
 def SmartConnect(protocol='https', host='localhost', port=443, user='root', pwd='',
                  service="hostd", path="/sdk",
-                 preferredApiVersions=None):
+                 preferredApiVersions=None, compress=True):
```

Comment:
Default behaviors should always be "whatever you did before" ... SoapStubAdapter currently defaults to compressed responses ... so this is actually correct.